_layouts/default.html

```diff
@@ -8,6 +8,16 @@
     <meta name="description" content="{% if page.excerpt %}{{ page.excerpt | strip_html | strip_newlines | truncate: 160 }}{% else %}{{ site.description }}{% endif %}">
     <link rel="stylesheet" href="/assets/stylesheets/application.css">
     <link rel="canonical" href="{{ page.url | replace:'index.html','' | prepend: site.baseurl | prepend: site.url }}">
+    {% if site.production %}
```

Comment:
This a custom GoStatic variable that is injected at deploy-time, correct syntax is:

`{% if site.environment == 'production' %}`